docs/codeql/codeql-overview/codeql-changelog/index.rst

```diff
@@ -0,0 +1,84 @@
+.. _codeql-changes:
+
+==================
+CodeQL change logs
+==================
+
+TODO: Write any kind of introduction here.
```

Comment:
Do we want to write an introduction here, or just delete this line? Potentially we could put here the information that's currently shown on line 12 of the individual articles?

Reply:
I don't have a strong opinion, happy to also add that line here. I would like to keep it in each of the docs in any case, so the info is easily available if someone is linked to a single changelog.